src/main/java/seedu/bookmark/ui/BookCard.java

```diff
@@ -89,6 +96,19 @@ protected void initialize(Book book, int displayedIndex) {
         book.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        String goalString = "Goal";
+        if (!book.hasGoal()) {
```

Comment:
Maybe you can extract this part out into another method?

Reply:
Also I think you forgot to remove the setting of the goal text/style from `DetailedBookCard`. Since it's being set here already and `DetailedBookCard` extends `BookCard` and will call this initialize method, there's no need to set it again, in fact `DetailedBookCard`  might still be using the old styling if you didn't remove it

Reply:
Also I think you need to remove the setting of the goal style and text from `DetailedBookCard`, otherwise the detailed view will still be using the old goal style